opentelemetry-sdk/tests/resources/test_resources.py

```diff
@@ -51,6 +52,13 @@ def test_create(self):
         self.assertIsInstance(resource, resources.Resource)
         self.assertEqual(resource.attributes, expected_attributes)
```

Comment:
Perhaps you might add another line like this?
```suggestion
        self.assertEqual(resource.attributes, expected_attributes)
        self.assertEqual(resource.schema_url, "")
```

Reply:
Also maybe a similar thing between lines 67-68? Especially if you figure out if OTELResourceDetector is able to somehow find a schema_url?

Reply:
Done!